apps/www/registry/new-york/ui/card.tsx

```diff
@@ -35,7 +35,7 @@ const CardTitle = React.forwardRef<
 >(({ className, ...props }, ref) => (
   <h3
     ref={ref}
-    className={cn("font-semibold leading-none tracking-tight", className)}
+    className={cn("max-w-80 overflow-hidden text-ellipsis max-lg:max-w-xl font-semibold leading-none tracking-tight", className)}
```

Comment:
@MohammedAslam106 did you mean to change this?

Reply:
Both new-york and default cards must be changed.

Reply:
I mean why the change to cards?

Reply:
Maybe you have seen images. For long text, it's breaking the card.

Reply:
Maybe even if we don't add it here. Users can change it according to their use case.

Reply:
Can we do this in a separate PR please?

Reply:
I guess this change is not related to the issue.